lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`ProjectReference` seems to be doing something different than what we want. I've tried adding such references and deleting one of built `.Tests` folders, and it didn't trigger rebuild, the corresponding test just failed.